src/main/resources/db/schema/SL-1876-PROPOSAL-hearing-part-change-of-columns.yaml

```diff
@@ -0,0 +1,88 @@
+databaseChangeLog:
+  - changeSet:
+        id: SL-1876-changes-to-hearing-part-table
+        author: lgurniewicz
+        changes:
+        # constraints drop
+          - dropForeignKeyConstraint:
+              baseTableName: hearing
```

Comment:
Shouldn't be hearing_part?

Reply:
yes it should, thankfully this file is not yet attached to dbchanges